packages/typescriptlang-org/lib/bootup/ingestion/createPlaygroundExamplePages.ts

```diff
@@ -64,14 +60,11 @@ export const createPlaygroundExamplePages = async (
     const exampleCodePath = path.join(appRoot, "playground-examples", "copy", rPath)
     const code = fs.readFileSync(exampleCodePath, "utf8")
 
-    const id = postLangPath
-      .split("/")
-      .slice(-1)[0]
-      .split(".")[0]
+    const id = postLangPath.split("/").slice(-1)[0].split(".")[0]
 
     const { inlineTitle, compilerSettings } = getCompilerDetailsFromCode(code)
     createPage({
-      path: newPagePath,
+      path: newPagePath + ".html",
```

Comment:
Prev: `https://www.typescriptlang.org/ja/play/typescript/language-extensions/types-vs-interfaces.ts`
Post: `https://www.typescriptlang.org/ja/play/typescript/language-extensions/types-vs-interfaces.ts.html`

I think it's safe to assume that search engines don't try read `.ts` or `.js` files.